### PR TITLE
Use better texts in window title

### LINKED ISSUE #513: Possible typo
https://github.com/FedoraQt/MediaWriter/blob/594ca9094e85426c3c0426a5566af35d339c189b/src/app/qml/DownloadPage.qml#L365

I'm not sure what it should be but it makes sense if I'm changing no to on.

### PR DESCRIPTION
**Pull request by grulja**
Fixes #513

@yarons does this make sense?

### CODE REVIEW

**Comment by yarons** [comment]
Failed "due to" or "over" sounds more natural but that's wonderful as it is.

**Comment by grulja** [comment]
> Failed "due to" or "over" sounds more natural but that's wonderful as it is.

Thanks. Fixed with your suggestion.

**Comment by bittin** [comment]
Will translate to Swedish once Weblate takes it up later today/tommorow

**Comment by yarons** [comment]
Sorry for all the mess guys, thank you!

**Comment by bittin** [comment]
Swedish updated

File: src/app/qml/DownloadPage.qml [hunk 1]
```diff
@@ -344,7 +344,7 @@ Page {
             }
             PropertyChanges {
                 target: mainWindow;
-                title: qsTr("Write Successful")
+                title: qsTr("Sucessfuly written")
```
Comment (piotrdrag) [inline src/app/qml/DownloadPage.qml:347]: Typo, should be `successfully`.

**Pull request merged**